docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -43,19 +41,18 @@ To complete this tutorial, you'll need:
 * An {es} cluster with hot and warm data tiers.
 
 ** {ess}:
-Elastic Stack deployments on {ess} include a hot tier by default. To add a warm
-tier, edit your deployment and click **Add capacity** for the warm data tier.
-+
+Elastic Stack deployments on {ess} automatically include hot and warm tiers. If needed, you can add additional warm capacity by editing your deployment and selecting **Add capacity** for the warm data tier.
++ 
 [role="screenshot"]
-image::images/ilm/tutorial-ilm-ess-add-warm-data-tier.png[Add a warm data tier to your deployment]
+image::images/ilm/tutorial-ilm-add-warm-data-tier.png[Add a warm data tier to your deployment]
```

Comment:
Is there a reason you changed this image link? As part of this the pngs should be updated to show the newer UI.